app-admin/supervisor/supervisor-3.3.4.ebuild

```diff
@@ -0,0 +1,83 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )	# py2 only
+# xml.etree.ElementTree module required.
+PYTHON_REQ_USE="xml"
+
+inherit distutils-r1 user
+
+MY_PV="${PV/_beta/b}"
+
+DESCRIPTION="A system for controlling process state under UNIX"
+HOMEPAGE="http://supervisord.org/ https://pypi.org/project/supervisor/"
+SRC_URI="mirror://pypi/${P:0:1}/${PN}/${PN}-${MY_PV}.tar.gz"
+
+LICENSE="repoze ZPL BSD HPND GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test"
+
+RDEPEND="
+	dev-python/meld3[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]
+"
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	doc? ( dev-python/sphinx[${PYTHON_USEDEP}] )
+	test? (
+		${RDEPEND}
+		dev-python/mock[${PYTHON_USEDEP}]
+	)
+"
+
+S="${WORKDIR}/${PN}-${MY_PV}"
+
+python_prepare_all() {
+	# https://wiki.gentoo.org/wiki/Project:Python/Namespace_packages
+	sed -i '/namespace_packages=/d' setup.py || die
+	# Upstream doesn't want to merge this
+	# https://github.com/Supervisor/supervisor/pull/1100
+	sed -i '1s/^/; vim: set ft=dosini:\n/' supervisor/skel/sample.conf || die
```

Comment:
Setting vim modelines ourselves is indeed questionable if upstream does not want to include it. Should we really keep it ? Especially now that we have a decent default configuration ?

Reply:
it's cosmetic, that is purely optional :) I'll delete it.